src/index.css

```diff
@@ -69,6 +69,14 @@ input:focus {
   border-radius: 5px;
   margin-bottom: 20px;
   word-wrap: break-word;
+  cursor: pointer;
```

Comment:
cursor pointer is not the property we'd want. post card is not a clickable or an action... it's just a card to view and items inside it are clickable actually